internal/cephfs/core/volume.go

```diff
@@ -199,16 +199,13 @@ func (s *subVolumeClient) GetSubVolumeInfo(ctx context.Context) (*Subvolume, err
 type operationState int64
 
 const (
-	unknown operationState = iota
-	supported
+	supported operationState = iota
 	unsupported
 )
 
 type localClusterState struct {
 	// set the enum value i.e., unknown, supported,
 	// unsupported as per the state of the cluster.
-	resizeState                 operationState
-	subVolMetadataState         operationState
 	subVolSnapshotMetadataState operationState
```

Comment:
please remove `subVolSnapshotMetadataState` as well?

can you also check do we have metadata support in `v16.0.0` as well?

Reply:
metadata & snapshot metadata support are added on pacific from v16.2.11 and on quincy it is from v17.2.1. Should we keep both these checks for some more time in that case, if someone uses versions lower than that?

Reply:
If that is a supported version in ceph we need to keep it as well. ie (< 16.2.11) and (<17.2.1)

Reply:
As per the docs 16.2.* and 17.2.* are still supported. So we need to keep these checks for some more time.
Will update support matrix as well to point to >=v16.2.0 for all the pacific versions.

Reply:
i believe in `ResizeSubVolume`  we can remove the check and also remove the call for createVolume. can you please check on that

Reply:
Yes, the support for it is present in all the active releases. Removed it now.